.github/workflows/R-test-coverage.yml

```diff
@@ -0,0 +1,50 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+name: test-coverage
+
+jobs:
+  test-coverage:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
NB: I have noticed that we have both `GITHUB_TOKEN` and `REPO_TOKEN` as organization level secrets. Is there a preference over which to use, for what, and why? Or are they just duplicates.

Reply:
IIRC, `REPO_TOPKEN` grants `pull` to our private repos. `GITHUB_TOKEN` is a GitHub builtin that is scoped to the current repo in question.

so REPO_TOKEN is needed if you're pulling in content from another (private) repo.